data/scenarios/Testing/378-objectives.yaml

```diff
@@ -0,0 +1,64 @@
+name: Complete two objectives in succession.
+description: |
+  First, get some trees; then, use them to build a harvester.
```

Comment:
I would not mind including this as part of the Place tutorial. :wink:

Reply:
I am including it as part of https://github.com/swarm-game/swarm/tree/multistep-tutorial .